src/sass/modules/_m-modal.scss

```diff
@@ -44,24 +44,24 @@
       background: transparent !important;
     }
   }
-}
 
-.va-modal button {
-  white-space: nowrap;
-} 
+  button {
+    white-space: nowrap;
+  } 
 
-.va-modal-close {
-  background-color: inherit;
-  color: rgba($color-base, .8);
-  font-size: 2.5rem;
-  padding: .5rem 1rem;
-  position: absolute;
-  margin: .25rem;
-  right: 0;
-  top: 0;
-  
-  &:hover {
+  &-close {
     background-color: inherit;
-    color: $color-base;
-  }
-} 
+    color: rgba($color-base, .8);
+    font-size: 2.5rem;
```

Comment:
Punting this one over to Vanguard. It looks like every button with a `usa-button-` prefix needs to have a `span` inside in order for it to render as bold. Can fix in this project later, but this is a global issue that affects the entire site.